sunspot/lib/sunspot/search/stats_row.rb

```diff
@@ -10,35 +10,43 @@ def initialize(data, facet = nil, value = nil) #:nodoc:
       end
 
       def min
-        data['min']
+        data['min'].to_i
       end
 
       def max
-        data['max']
+        data['max'].to_i
       end
 
       def count
-        data['count']
+        data['count'].to_i
       end
 
       def sum
-        data['sum']
+        data['sum'].to_i
       end
 
       def missing
-        data['missing']
+        data['missing'].to_i
+      end
+
+      def sumsq
+        sum_of_squares
       end
 
       def sum_of_squares
-        data['sumOfSquares']
+        !data['sumOfSquares'].nil? ? data['sumOfSquares'] : data['sumsq'].to_i
+      end
+
+      def avg
+        mean
       end
 
       def mean
-        data['mean']
+        !data['mean'].nil? ? data['mean'] : data['avg'].to_i
```

Comment:
same as above. I dont see `avg` function [here](https://lucene.apache.org/solr/guide/6_6/the-stats-component.html)
Please explain